audio/fx2lights.h

```diff
@@ -0,0 +1,42 @@
+/**
```

Comment:
Because it is used by the audio firmware (firmware.c) and provides an overlay to the lights.h header that works with the FX2 dev board (and in theory the opsis too). May be better as an update to the lights.h header directly...